src/main/java/Sb2.java

```diff
@@ -131,5 +132,24 @@ public int getScriptCountForSprite(String spriteName) {
     public int[] getScriptLengthsForSprite(String spriteName) {
         return sprites.getScriptLengthsForSprite(spriteName);
     }
-}
+    /**
+     * Gets the number of global variables within the stage object.
+     * @return numVariables The number of global variables
+     */
+    public int getGlobalVariableCount() {
```

Comment:
Why is this set up as a method that computes global state? Seems like a strange architecture.